packages/core/state/interaction/logics.ts

```diff
@@ -1,4 +1,5 @@
 import { isEmpty, sumBy, throttle, uniq, uniqueId } from "lodash";
+import { isAbsolute } from "path";
```

Comment:
isn't path a `node` module aka not available for web deploys?

Reply:
ah rats I always forget about this. (And didn't test it on web since I was using a remote workstation)